radix-engine-interface/src/data/types/manifest_blob.rs

```diff
@@ -0,0 +1,56 @@
+use sbor::rust::convert::TryFrom;
+#[cfg(not(feature = "alloc"))]
+use sbor::rust::fmt;
+use sbor::rust::vec::Vec;
+use sbor::*;
+use utils::copy_u8_array;
+
+use crate::crypto::Hash;
+use crate::data::*;
+use crate::scrypto_type;
+
+#[derive(Debug, Clone, PartialEq, Eq, Hash)]
+pub struct ManifestBlob(pub Hash);
```

Comment:
What do you think about `ManifestBlobHash` or `ManifestBlobRef` instead? Because it's not technically the blob itself?

Reply:
Happy to change to `ManifestBlobRef`. Will do as part of this PR.